.github/actions/affected/action.yml

```diff
@@ -31,7 +31,11 @@ runs:
         if [[ "$affected" = *"macos"* ]]; then
           echo 'macos=true' >> $GITHUB_OUTPUT
         fi
+        if [[ "$affected" = *"visionos"* ]]; then
+          echo 'visionos=true' >> $GITHUB_OUTPUT
+        fi
         if [[ "$affected" = *"windows"* ]]; then
           echo 'windows=true' >> $GITHUB_OUTPUT
         fi
+        echo $affected
```

Comment:
Even though visionOS is printed here, CI doesn't seem to want to build the visionOS app

Reply:
Aaaand fixed!